llvm/test/tools/llvm-mca/RISCV/GenericOOO/atomic.s

```diff
@@ -0,0 +1,601 @@
+# NOTE: Assertions have been autogenerated by utils/update_mca_test_checks.py
+# RUN: llvm-mca -mtriple=riscv64 -mattr=+rva23u64,+zabha -mcpu=generic-ooo --all-stats -iterations=1 < %s | FileCheck %s
```

Comment:
nit: `--all-stats` shows many MCA internals like PRF and ROB utilization, I'm a little concerned that such info is more volatile and more importantly, is not directly related to scheduling model. Could we just use the default view?